src/backend/interfaces/NATS/py.client/caliopen_nats/delivery.py

```diff
@@ -7,19 +7,23 @@
 import datetime
 import pytz
 from caliopen_storage.exception import NotFound
-from caliopen_main.message.core import RawMessage
+from caliopen_main.message.core import RawMessage, MessageExternalRefLookup
 from caliopen_main.message.objects.message import Message
+from caliopen_main.message.store.message import ModelMessageExternalRefLookup
 from caliopen_pi.qualifiers import UserMessageQualifier, UserDMQualifier
 
 log = logging.getLogger(__name__)
 
 
 class UserMessageDelivery(object):
-    """User message delivery processing."""
+    user = None
```

Comment:
do not define here variables that will be used in the __init__

Declaring this way make variables common to all instance of this class.

Reply:
fix with commit #7c8c9aa9